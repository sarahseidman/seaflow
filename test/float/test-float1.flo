float i = 7.0;
float j = 12.0;
float k = i + j;

printf(i);
printf(j);
printf(k);
