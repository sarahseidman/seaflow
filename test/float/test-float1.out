7.000000
12.000000
19.000000
